api/src/main/java/org/openmrs/module/ugandaemrreports/library/CommonDimensionLibrary.java

```diff
@@ -405,6 +405,114 @@ public CohortDefinitionDimension getTxNewAgeGenderGroup() {
 
     }
 
+    public CohortDefinitionDimension getNewTxCurrAgeGenderGroup() {
```

Comment:
@dbaluku Why name it to newTxCurrAgeGenderGroup() why are we keeping the old one since its not used anymore